master/static/migrations/20230920163257_add-allocation-accelerators-table.tx.up.sql

```diff
@@ -0,0 +1,7 @@
+ CREATE TABLE public.allocation_accelerators (
+    container_id text NOT NULL PRIMARY KEY,
+    allocation_id text NOT NULL REFERENCES public.allocations(allocation_id) ON DELETE CASCADE,
+    node_name text NOT NULL,
```

Comment:
we should index on allocation_id at least.

Reply:
Done